<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>733</width>
      <height>367</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Principal</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre de Usuario:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nombre_usuario"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="guardar_nom">
        <property name="text">
         <string>Guardar Nombre</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="primera_civ">
        <property name="text">
         <string>Primera Civilizacion</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="4" colspan="2">
       <widget class="QPlainTextEdit" name="pantalla"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="ultima_civ">
        <property name="text">
         <string>Ultima Civilizacion</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="ordenar">
        <property name="text">
         <string>Ordenar</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="resumen">
        <property name="text">
         <string>Resumen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Agregar</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nombre de Civilizacion:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nombre_civ_agr"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Posicion en X:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="pos_x_agr"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Posicion en Y:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="pos_y_agr"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Posicion de Civilizacion:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="pos_agr"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="agregar">
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Inicializar</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Nombre de Civilizacion:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nombre_civ_ini"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Posicion en X:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="pos_x_ini"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Posicion en Y:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="pos_y_ini"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Numero de Civilizaciones:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="numero_civ_ini"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="inicializar">
        <property name="text">
         <string>Inicializar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Eliminar</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Nombre de Civilizacion a eliminar:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nombre_eliminar"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="eliminar">
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Buscar</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre de Civilizacion:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombre_buscar">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Buscar">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>271</width>
        <height>211</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="3" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Salud</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="genero_ald_agr"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Nombre</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Edad</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Genero</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nom_ald_agr"/>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="agr_ini">
          <property name="text">
           <string>Agregar al Inicio</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="agr_fin">
          <property name="text">
           <string>Agregar al Final</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="edad_ald_agr"/>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="salud_ald_agr"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Eliminar</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>231</width>
          <height>131</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_12">
         <attribute name="title">
          <string>Nombre</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="1" column="1">
           <widget class="QPushButton" name="eliminar_ald_nom">
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nombre_eli_ald"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Nombre:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Salud</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Salud:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="edad_elim"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="eliminar_ald_sal">
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Edad</string>
         </attribute>
         <widget class="QLabel" name="label_20">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>20</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Eliminara a los aldeanos con</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_21">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>40</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>edad mayor o igual a 60.</string>
          </property>
         </widget>
         <widget class="QPushButton" name="eliminar_ald_eda">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>70</y>
            <width>186</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Eliminar</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>160</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Bajo que criterio quieres eliminar?</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Ordenar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="2" column="0">
         <widget class="QPushButton" name="ord_ald_nom">
          <property name="text">
           <string>Nombre (ascendente)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="ord_ald_eda">
          <property name="text">
           <string>Edad (descendente)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="ord_ald_sal">
          <property name="text">
           <string>Salud (descendente)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Bajo que criterio quieres ordenar?</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>Mostrar</string>
       </attribute>
       <widget class="QPushButton" name="actualizar_tabla">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>100</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Actualizar</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Presiona para actualizar la tabla.</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPlainTextEdit" name="pantalla2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>110</y>
        <width>351</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Administracion de Aldeanos:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
